### PR TITLE
implement fancy updater

### PR DESCRIPTION
**Pull request by maghuro**
and change all FlexQoS to var

### CODE REVIEW

**Comment by poorfish** [comment]
1111

**Comment by poorfish** [comment]
> 1111

2222

File: flexqos.sh [hunk 1]
```diff
@@ -39,7 +39,7 @@ release="2020-07-11"
 # Global variables
 SCRIPTNAME="flexqos"
 SCRIPTNAME_FANCY="FlexQoS"
-GIT_REPO="https://raw.githubusercontent.com/dave14305/FlexQoS"
+GIT_REPO="https://raw.githubusercontent.com/dave14305/$SCRIPTNAME_FANCY"
```
Comment (poorfish) [inline flexqos.sh:42]: 33333

Comment (poorfish) [reply flexqos.sh:42]: 4444

Comment (poorfish) [reply flexqos.sh:42]: > 33333

5555

**Comment by maghuro** [comment]
rebased

**Pull request closed without merging**